js/src/components/google-combo-account-card/claim-ads-account/claim-ads-account.js

```diff
@@ -41,47 +42,48 @@ const ClaimAdsAccount = () => {
 		}
 	}, [ hasAccess, step, upsertAdsAccount ] );
 
+	const shouldClaimGoogleAdsAccount = Boolean(
```

Comment:
If we're moving this from `ConnectedAccountsActions` to inside this component, then we also need to update the `checkUpdatedAdsAccountStatus()` callback so that is stops calling the `fetchGoogleAdsAccountStatus()` whenever `shouldClaimGoogleAdsAccount` is `false`. Otherwise, the API polling will continue even after the account is claimed.

Reply:
Made this change and pushed.